.pre-commit-config.yaml

```diff
@@ -20,7 +20,8 @@ repos:
     hooks:
       - id: isort
         name: isort (python)
-  - repo: https://github.com/PyCQA/flake8
-    rev: 6.0.0
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    # Ruff version.
+    rev: v0.0.275
```

Comment:
Can we use `main` to get the latest version (I presume GHA will get latest ruff as well)?

Reply:
pre-commit prefers explicit versions and you can then periodically run `pre-commit autoupdate`.